src/components/structure/interacting/search-result-navigation/search-result-navigation.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useContext } from 'react';
+import { observer } from 'mobx-react-lite';
+
+import translations from './translations.json';
+import './search-result-navigation.scss';
+
+import StoreContext, { GlobalSearchEntityType } from '../../../../store/StoreContext';
+
+type Translations = {
+  de: Record<string, string>
+};
+
+const range = 6;
+
+const SearchResultNavigation: FC<Props> = ({
+  hits = 0,
+  size = 0,
+  from = 0
+}) => {
+  const useTranslation = (_: string, translations: Translations) => ({ t: (key: string, _?: Record<string, string>) => translations.de[key] });
+  const { t } = useTranslation('Navigation', translations);
+  const { globalSearch } = useContext(StoreContext);
+  const pages = Math.ceil(hits / size);
```

Comment:
An sich spricht nichts dagegen viele dieser Werte und Zustände (`pages`, `pos`, `firstIsActive`, `lastIsActive`, `getChunks`) in der Komponente zu halten.
Frage mich aber, ob es vllt. sinnvoller sein könnte, diese Werte und Zustände in den Store zu verschieben und in Computed-Values zu überführen, da es sich größtenteils um Daten handelt, die von Store-State (`hits`, `size`, `from`) abgeleitet werden.